src/cascadia/TerminalApp/SuggestionsControl.xaml

```diff
@@ -12,6 +12,7 @@
              xmlns:model="using:Microsoft.Terminal.Settings.Model"
              xmlns:mtu="using:Microsoft.Terminal.UI"
              xmlns:mux="using:Microsoft.UI.Xaml.Controls"
+             MinWidth="256"

```

Comment:
Does it need to be this much, or can we let the width get adjusted dynamically as a percentage of the page width? Do we need a `MaxWidth`?

Reply:
Ah, that was an unrelated nit that snuck in accidentally. IMO this makes it feel a lot smoother, since the width doesn't snap back and forth. The placeholder text makes the menu open at like, 252px or something, but then filtering the menu usually immediately makes it quite a bit narrower. This keeps the width a bit more consistent